beetsplug/bpd/__init__.py

```diff
@@ -1404,16 +1404,28 @@ def cmd_list(self, conn, show_tag, *kv):
         filtered by matching match_tag to match_term.
         """
         show_tag_canon, show_key = self._tagtype_lookup(show_tag)
+        if len(kv) == 1:
+            if show_tag_canon == 'Album':
+                # If no tag was given, assume artist.
```

Comment:
It might be a good idea to mention here that this is a non-standard, seemingly undocumented behavior of MPD that some clients (namely, MALP) rely on to list the albums for a given artist.

Reply:
Ok, done!